src/industry_gui.cpp

```diff
@@ -1035,6 +1052,11 @@ class IndustryViewWindow : public Window
 				ShowIndustryCargoesWindow(i->type);
 				break;
 			}
+
+			case WID_IV_GRAPH: {
+				ShowIndustryProductionGraph(this->window_number);
+				break;
+			}
```

Comment:
No need for a `{ }` block for this switch case.

Reply:
Fixed.